packages/module/src/DetailsPageHeader/DetailsPageHeader.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  Split,
+  SplitItem,
+  TextContent,
+  Text,
+  TextVariants,
+  Label,
+} from '@patternfly/react-core';
+import React from 'react';
+import {
+  ActionButtonProps,
+  ActionButtons,
+  ActionMenu,
+  ActionMenuProps,
+  BreadcrumbProps,
+  Breadcrumbs,
+} from './utils';
+import '@patternfly/react-styles/css/utilities/Spacing/spacing.css';
+
+export interface PageHeading {
+  /** Title for page heading */
+  title: string;
```

Comment:
This was optional, but is now required since `obj` was removed from `DetailsPage.tsx`.